src/main/java/org/icgc_argo/workflow/search/model/common/Run.java

```diff
@@ -72,24 +70,10 @@
   /** The command line that was executed */
   private String commandLine;
 
-  private EngineParameters engineParameters;
+  private RunEngineParams engineParameters;
```

Comment:
RunEngineParams? What is the justification for the rename?

Reply:
hmm, thats a good point I don't know why I called it that...changing back to EngineParameters